File: 2015/common/src/blank.rs
use common;
const INPUT_FILE: &str = "input/dayXX.txt";

fn main() {
    pt1();
    pt2();
}

fn pt1() {
    // Test
    assert_eq!(task1(), ());

    // Solution
    let input: Vec<String> = common::read_file(INPUT_FILE);
    let input: String = input.first().unwrap().to_string();

    let ans = task1(input);
    println!("Task 1 solution: {ans}");
}

fn pt2() {
    // Test
    assert_eq!(task2(), ());

    // Solution
    let input: Vec<String> = common::read_file(INPUT_FILE);
    let input: String = input.first().unwrap().to_string();

    let ans = task2(input);
    println!("Task 2 solution: {ans}");
}

fn task1(input: String) -> i32 {}

fn task2(input: String) -> i32 {}

File: 2016/blank.py
from utils import *

FILE = "input/dayXX.txt"


def main():
    pt1()
    pt2()


def pt1():
    # Test
    test(1, 1)

    # Solution
    input = read_file(FILE)
    print(f"Task 1 solution: {task1(input)}")


def pt2():
    # Test
    test(1, 1)

    # Solution
    input = read_file(FILE)
    print(f"Task 2 solution: {task2(input)}")


#########################################################


def task1(input):
    pass


def task2(input):
    pass


if __name__ == "__main__":
    # benchmark(main())
    main()
